### PR TITLE
docs(commercial): Add new docs for commercial vs core

### PR DESCRIPTION
**Pull request by adnanrahic**
This PR adds new docs onboarding for the commercial launch.

Preview: https://651b484c32b4131be47440c6--lively-cassata-a55b4a.netlify.app/

## Changes

-

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

**Comment by kdhamric** [comment]
Other than a couple TBD at https://651b484c32b4131be47440c6--lively-cassata-a55b4a.netlify.app/getting-started/installation#install-the-tracetest-agent for docker and kubernetes, looking good!

**Review by kdhamric — APPROVED** [review]
Other than a couple TBD at https://651b484c32b4131be47440c6--lively-cassata-a55b4a.netlify.app/getting-started/installation#install-the-tracetest-agent for docker and kubernetes, looking good!

**Comment by adnanrahic** [comment]
Thanks @kdhamric !

Added an update for those TBD items.

Preview: https://651bbef26346fc3c3465bbea--lively-cassata-a55b4a.netlify.app/getting-started/installation#install-the-tracetest-agent

File: docs/docs/core/deployment/kubernetes.mdx [hunk 1]
```diff
@@ -89,4 +89,4 @@ kubectl apply -f ./tracetest-kubernetes-manifests.yaml
   </TabItem>
 </Tabs>
 
-For more configuration options, read more in the [configuration docs](../configuration/overview.md).
+For more configuration options, read more in the [configuration docs](/configuration/overview.md).
```
Comment (jfermi) [inline docs/docs/core/deployment/kubernetes.mdx:92]:
```suggestion
For more configuration options, visit the [configuration docs](/configuration/overview.md).
```

File: docs/docs/getting-started/open.mdx [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+image: https://res.cloudinary.com/djwdcmwdz/image/upload/v1689693872/docs/Blog_Thumbnail_28_ugy2yy.png
+---
+
+This page showcases opening the Tracetest Web UI and creating a test against the [sample Pokeshop API](../live-examples/pokeshop/overview.md).
+
+Once you've installed Tracetest, as explained in the [installation guide](./installation.mdx), the Tracetest Agent is running on `localhost` ports `4317` and `4318`. You then access the Tracetest Web UI on [`app.tracetest.io`](https://app.tracetest.io). Here's what will greet you after a fresh install.
+
+![Landing page Tracetest](https://res.cloudinary.com/djwdcmwdz/image/upload/v1688474565/docs/screely-1688474539641_kbhvvc.png)
+
+## Start the sample Pokeshop API
```
Comment (jfermi) [inline docs/docs/getting-started/open.mdx:20]:
```suggestion
## Start the Sample Pokeshop API
```

File: docs/docs/getting-started/open.mdx [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+
+This will populate a sample API test against a POST endpoint in the Pokeshop app demo. Clicking **Create & Run** will save and trigger the test.
+
+![API test against POST endpoint](https://res.cloudinary.com/djwdcmwdz/image/upload/v1696253160/docs/screely-1696253153066_khjz8y.png)
+
+### Trigger
+
+You'll be redirected to the test page where you can see four tabs and depending on which one you select you'll get access to:
+
+- Test trigger and results
```
Comment (jfermi) [inline docs/docs/getting-started/open.mdx:137]:
```suggestion
- Test trigger and results.
```

File: docs/docs/getting-started/open.mdx [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+- The entire distributed trace and trace analysis
+- Test specification and assertions
```
Comment (jfermi) [inline docs/docs/getting-started/open.mdx:139]:
```suggestion
- Test specification and assertions.
```

File: docs/docs/getting-started/open.mdx [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+- How to automate the test
```
Comment (jfermi) [inline docs/docs/getting-started/open.mdx:140]:
```suggestion
- How to automate the test.
```

File: docs/docs/getting-started/open.mdx [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+
+:::note
+To view the in-depth guide on test results, [check out this docs page](/web-ui/test-results)
```
Comment (jfermi) [inline docs/docs/getting-started/open.mdx:143]:
```suggestion
To view the in-depth guide on test results, [check out this docs page](/web-ui/test-results).
```

**Review by jfermi — COMMENTED** [review]
A few editing suggestions.

**Comment by adnanrahic** [comment]
Nice! Applied. I'm happy to merge this right away if everyone agrees.

**Comment by adnanrahic** [comment]
Preview: https://651d4c91d7f20b443dcd25fb--lively-cassata-a55b4a.netlify.app/core/getting-started/overview

**Pull request merged**